README.md

```diff
@@ -101,6 +101,16 @@ try {
 }
 ```
 
+## Core Functionality
+
+- **Send Errors with Stack Trace** 📤
```

Comment:
nit: We used emojis before the text above, should not we do it the same way here?

like
```
📤 ** Send Errors with Stack Trace** 
```